README.md

```diff
@@ -8,7 +8,7 @@ The IBM Power Virtual Systems Container Storage Interface (CSI) Driver provides
 # CSI Specification Compatibility Matrix
 | PowerVS CSI Driver | Kubernetes | CSI | Golang |
 | ----------------------------- | ----------- | -------- | -------- |
-| main | 1.26 | 1.7.0 | 1.19 |
+| main | 1.28 | 1.7.0 | 1.20 |
```

Comment:
I think 1.28 uses 1.21 golang if I'm not wrong! can you crosscheck once?

Reply:
https://github.com/kubernetes/kubernetes/blob/v1.28.1/go.mod#L9 `go 1.20`

Reply:
@mkumatag with from 1.27.3 we are having dependency issues.
https://github.com/kubernetes/client-go/issues/1269
https://github.com/kubernetes/client-go/issues/1075

Hence I am trying to use 1.28 k8s which is stable.

Reply:
Can you please document the issues you faced so that it will be helpful for us if we comeback later to understand why we jumped directly to 1.28 release?

Reply:
Yes, but the the location you need to refer is - https://github.com/kubernetes/kubernetes/blob/release-1.28/.go-version#L1
>https://github.com/kubernetes/kubernetes/blob/v1.28.1/go.mod#L9 go 1.20

Reply:
created https://github.com/kubernetes-sigs/ibm-powervs-block-csi-driver/issues/451